packages/dds/ordered-collection/src/consensusOrderedCollection.ts

```diff
@@ -217,23 +217,16 @@ export class ConsensusOrderedCollection<T = any>
         return tree;
     }
 
-    protected isActive() {
```

Comment:
I'd not change this behavior here (in same PR).

Reply:
Reverted in latest.